cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/partitioned/PartitionedFileSetDataset.java

```diff
@@ -71,6 +71,8 @@
   protected static final byte[] RELATIVE_PATH = { 'p' };
   protected static final byte[] FIELD_PREFIX = { 'f', '.' };
   protected static final byte[] METADATA_PREFIX = { 'm', '.' };
+  // also used as the rowkey prefix of the creation time index
+  protected static final byte[] CREATION_TIME = { 'c', '.' };
```

Comment:
I was originally having another table, as the index on the creation time, but removed that from this PR.
So, removed the trailing dot, as its not necessary.